content/en/docs/contributing/release-process.md

```diff
@@ -270,10 +270,11 @@ A patch release contains critical bug fixes for the project.  They are managed o
 an ad-hoc basis, and should only be required when critical bugs/regressions are
 found in the release.
 
-We will only perform patch release for the **current** version of cert-manager.
+We will only perform patch release for the **current** and **last** minor
```

Comment:
Suggestion: perhaps we could use the word 'previous' instead of 'last' here or just describe them as `n`, `n-1` ? 'last' is confusing as we seem to sometimes use it to describe 'latest'.